code/modules/mob/living/carbon/human/human_life.dm

```diff
@@ -71,6 +74,36 @@
 		if(A.fast_despawn)
 			force_cryo_human(src)
 
+/mob/living/carbon/human/proc/handle_light_adjustment()
+	var/turf/T = get_turf(src)
+	var/brightness = T.get_lumcount() // Will return a number from 0 to 1, with 1 being the brightest
```

Comment:
This needs a nullcheck
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/2d13210e-6250-4a8d-a02e-afffd4ef58b1)